### ISSUE TITLE
Feature request: Spellcheck

### ISSUE THREAD

**Issue by nathansamson** [sonnyp/Commit#90]
Title says it all. Builtin spellcheck so you don't make typo's...

**Comment by sonnyp** [comment]
https://gitlab.gnome.org/GNOME/gtk/-/issues/3814

**Comment by nathansamson** [comment]
Could it be integrated with this?

https://blogs.gnome.org/chergert/2023/07/12/spellchecking-for-gtk-4/

**Comment by sonnyp** [comment]
yes https://gitlab.gnome.org/chergert/libspelling/#in-javascript

**Comment by nathansamson** [comment]
Let me rephrase my question :)

Are you aware of this new spelling library, and do you have plans to integrate it (or are willing to accept contributions that use it)

**Comment by sonnyp** [comment]
I'm aware of it and Commit should make use of it :+1: but I don't have time to work on it.

I'd be very happy to review/help if someone is willing to give it a try.

This is what needs to be done

1. Add libspelling module to the Flatpak manifest
2. Integrate it, make sure to use ESM to import it `import Spelling from 'gi://Spelling';`

https://gitlab.gnome.org/chergert/libspelling/-/tree/main#in-javascript
